BEER tutorial

EyalLavi opened the pull request:
Fixes #151

EyalLavi committed:
Update tutorial.rst

alexandre-rouxel commented on docs/tutorial.rst:
```diff
@@ -250,3 +250,109 @@ Word Error Rate variants
+
+Consider for example this sentence in the reference transcript: 'The European Union headquarters'. If engine A returns 'The European onion headquarters' and engine B returns 'The European Union headache', the Word Error Rate would be similar for both engines since in both cases one word was transcribed inaccurately. But engine B should be 'rewarded' for preserving the phrase 'European Union'. The ...
+
+Another use for this metric is compensating for distortions of WER that are caused by normalization rules. For example, you may convert both reference and hypothesis transcripts to lower case or remove punctuation marks so that they don't affect the WER. In this case, the distinction between 'Theresa May' and 'Therese may' is lost. But you can instruct BenchmarkSTT to score higher the engine that ...
```
'Theresa may'

aro-max replied:
the distinction between 'Theresa May' and 'Theresa may' is lost.

aro-max commented on docs/tutorial.rst:
```diff
@@ -250,3 +250,109 @@ Word Error Rate variants
+
+   {"Theresa May" : 0.5, "Abigail" : 0.5, "EU": 0.75, "Griffin" : 0.5, "I" : 0.25}
+
+We'll also tell BenchmarkSTT to normalize the reference and hypothesis file by lowercasing both. We do this in the ``config.conf`` file: ::
```
We'll also tell BenchmarkSTT to normalize the reference and hypothesis file but without lowercasing both. We do this in the ``config.conf`` file: ::

aro-max committed:
typo : change the distinction between 'Theresa May'
and 'Theresa may' is lost.
change : l 328
We'll also tell BenchmarkSTT to normalize the reference and hypothesis
file but without lowercasing both. We do this in the config.conf
file: ::
Add some space between equations

aro-max commented:
I merged a new pull request with minor changes on the BEER tutorial.
